stratify/_conservative.pyx

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+cimport numpy as np
+
+
+cdef calculate_weights(np.ndarray[np.float64_t, ndim=2] src_point,
+                       np.ndarray[np.float64_t, ndim=2] tgt_point):
+    # src_min    src_max
+    #    |----------|                 : Source
+    #       tgt_min     tgt_max
+    #          |------------|         : Target
+    #    |----------|                 : Delta (src_max - src_min)
+    #          |----|                 : Overlap (between src & tgt)
+    #    weight = overlap / delta
```

Comment:
I really like this diagram - it provides the key concept and several definitions very succinctly.